File: solutions/python/rna-transcription/1/rna_transcription.py
def to_rna(dna_strand):
    res=''
    for dna in dna_strand:
        if dna=='G':
            res+='C'
        elif dna=='C':
            res+='G'
        elif dna=='T':
            res+='A'
        elif dna=='A':
            res+='U'
        else:
            pass
    
    return res       

File: solutions/javascript/rna-transcription/1/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const toRna = (sequence) => {
  let res = '';
  for(var i = 0; i < sequence.length; i++) {
    switch(sequence[i]) {
      case 'G':
        res+='C';
        break;
      case 'C':
        res+='G';
        break;
      case 'T':
        res+='A';
        break;
      case 'A':
        res+='U';
        break;
    }
  }
  return res;
};
